platform/engineering/collab-cycle/architecture-intent-template.md

```diff
@@ -0,0 +1,79 @@
+# Architecture Intent Template
+
+__*** DRAFT *** DRAFT *** DRAFT ***__
+
+The Architecture Intent meeting helps your team build a solution that meets Veteran-facing Service Platform (VSP) engineering standards and lowers the potential for launch-blocking issues later in the development cycle.
+
+## What is the purpose of Architecture Intent meeting?
+
+The Architecture Intent meeting is less a formal presentation and more of a discussion.  It not only provides OCTO-DE and Platform with an early understanding of the product/feature your team wants to build, but is also an opportunity to collaborate and provide feedback on the intended implementation and surface any adjustments needed to meet VSP engineering standards. The focus is on making sure your code meets user needs within the constraints of the platform your building on.
+
+## When to schedule an Architecture Intent meeting?
+
+You should schedule an Architecture Intent meeting if any of these apply:
+
+- You have questions about one or more points on the template below.
+- You're having trouble locating or getting technical info from other stakeholders or system owners.
+- You're using an architecture pattern not currently found on the va.gov platform.
+- You plan to use a new technology, library or dependency.
+- You're integrating with a new system or API, inside or outside of the VA.
+- You need to coordinate your change with other teams or your change will add work or responsiblity to other teams.
```

Comment:
This one would cover virtually all work on mobile, is the intent to review a lot of small items or just large more far-reaching ones? Sometimes small API or UI changes require coordination with upstream teams and services, so I could see this requirement being hit _a lot_ to where it could back up the system.

I see this also going in the other direction when any web team (who doesn't have a mobile engineer) adds a small feature that also needs to be added to their mobile version, and the flagship team is tasked with doing that work. Do we want to insert a collab cycle meeting into that?

Reply:
Great points.  I'm trying to find a sweet spot for two cases: (1) new teams or stuck teams need help, and (2) major changes.  Small to medium sized routine changes don't need to go through this.  I've rewritten this point to:
```
- You're change requires complex coordination across teams or you need support to coordinate across teams.
 ```
Definitely open to refining this further.